src/applications/proxy-rewrite/redirects/crossDomainRedirects.json

```diff
@@ -433,5 +433,50 @@
     "domain": "www.benefits.va.gov",
     "src": "/compensation/resources-rates-read-dic.asp",
     "dest": "/disability/survivor-dic-rates/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/veteran_borrowers.asp",
+    "dest": "/housing-assistance/home-loans/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/purchasecashout.asp",
+    "dest": "/housing-assistance/home-loans/loan-types/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/purchaseco_buy_process.asp",
+    "dest": "/housing-assistance/home-loans/home-buying-process/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/nadl.asp",
+    "dest": "/housing-assistance/home-loans/loan-types/native-american-direct-loan/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/irrrl.asp",
+    "dest": "/housing-assistance/home-loans/loan-types/interest-rate-reduction-loan/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/purchaseco_eligibility.asp",
+    "dest": "/housing-assistance/home-loans/eligibility/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/HOMELOANS/purchaseco_certificate.asp",
+    "dest": "/housing-assistance/home-loans/how-to-apply/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/HOMELOANS/resources_payments.asp",
```

Comment:
^^ https://github.com/department-of-veterans-affairs/vets-website/pull/11735/files#r382142303